packages/react-inbox/src/components/Messages/index.tsx

```diff
@@ -67,16 +71,21 @@ const Messages: React.FunctionComponent<InboxProps> = ({
             renderMessage(message)
           ) : (
             <Message key={message.messageId} {...message} />
-          ),
+          )
         )}
         {isLoading && <Loading />}
         {!isLoading && messages.length === 0 && (
           <Empty>You have no notifications at this time</Empty>
         )}
-        {!isLoading && messages.length && !startCursor && (<PaginationEnd title="End Of The Road" />) }
+        {!isLoading && messages.length > 5 && !startCursor && (
```

Comment:
im not too sure about this logic here, but i think we only want to show the end of the road if we have > 5 messages